src/post/types.ts

```diff
@@ -1,17 +1,16 @@
-export interface Post {
+export interface PostBase {
```

Comment:
¿Por qué la exportáis?

Reply:
ha sido acto reflejo, no habia pensado que el PostBase no lo uso fuera